src/wayscript/utils.py

```diff
@@ -0,0 +1,52 @@
+import os
+import string
+
+import requests
+
+from . import settings
+
+
+def get_process_execution_user_token():
+    """Return the auth token of the user this process is executing on behalf of"""
+    token = os.environ["WAYSCRIPT_EXECUTION_USER_TOKEN"]
+    return token
+
+
+def get_process_uuid():
+    """Return uuid of current container execution"""
+    process_uuid = os.environ["WAYSCRIPT_PROCESS_UUID"]
+    return process_uuid
+
+
+class WayScriptClient:
+
+    def __init__(self, *args, **kwargs):
+        """Init a wayscript client"""
+        self.session = requests.Session()
+        self.session.headers["authorization"] = get_process_execution_user_token()
+    
+    def _get_url(self, subpath: str, route: str, template_args: dict=None):
```

Comment:
Would love to have better words here than "subpath" and "route". It's like "top level route" and "sub route", almost.